### PR TITLE
REP-3273 SLF4JHttpLogging filter conversion and refactor

### PR DESCRIPTION
**Pull request by dmnjohns**

### CODE REVIEW

**Comment by dmnjohns** [comment]
I have to say, converting this filter to Scala really cleaned things up. At least, in my opinion.

**Comment by dmnjohns** [comment]
Those failures. Looking into it.

File: repose-aggregator/components/filters/slf4j-http-logging/src/main/scala/org/openrepose/filters/slf4jlogging/Slf4jLoggerWrapper.scala [hunk 1]
```diff
@@ -17,32 +17,11 @@
...
-
-    public String getFormatString() {
-        return formatString;
-    }
-
-    public Logger getLogger() {
-        return logger;
-    }
-}
+case class Slf4jLoggerWrapper(logger: Logger, formatString: String) {
```
Comment (adrianjgeorge) [inline repose-aggregator/components/filters/slf4j-http-logging/src/main/scala/org/openrepose/filters/slf4jlogging/Slf4jLoggerWrapper.scala]: Doesn't seem like it's worth it to keep this as a separate class at this point.

Comment (dmnjohns) [inline repose-aggregator/components/filters/slf4j-http-logging/src/main/scala/org/openrepose/filters/slf4jlogging/Slf4jLoggerWrapper.scala]: You mean as it's own file? I can totally move it.

**Pull request merged**